modules/virt-checking-cluster-dpdk-readiness.adoc

```diff
@@ -90,7 +106,56 @@ roleRef:
 $ oc apply -n <target_namespace> -f <dpdk_roles>.yaml
```

Comment:
On the same note - `dpdk_roles` should be renamed to something that reflects all the resources it covers, not only the `Roles`.